src/adapters/supabase/helpers/tables/access.ts

```diff
@@ -0,0 +1,75 @@
+import { SupabaseClient } from "@supabase/supabase-js";
+import { Database } from "../../types/database";
+import { GitHubNode } from "../../types/github";
+import { Super } from "./super";
+import { UserRow } from "./user";
+import { Context } from "../../../../types/context";
+import { Comment } from "../../../../types/github";
+
+type AccessRow = Database["public"]["Tables"]["access"]["Row"];
+type AccessInsert = Database["public"]["Tables"]["access"]["Insert"];
+type UserWithAccess = UserRow & { access: AccessRow[] };
+
+type AccessData = {
+  user_id: number;
+  multiplier: number;
+  multiplier_reason: string;
+  node_id: string;
+  node_type: string;
+  node_url: string;
+};
+
+export class Access extends Super {
```

Comment:
access table is for setting who can change specific labels types like time, priority, price and it's scoped per repository?

Reply:
Yes I vaguely recall this is implemented as saving a json object. Key value pairs for any arbitrary label that any user has permission to change, with the intent that projects should be able to customize the names of the priority labels and the time labels, to generate pricing.